operations/finance.md

```diff
@@ -295,7 +304,8 @@ Returns all accounting items of the enterprise that were consumed \(posted\) or
 
 | Property | Type | Contract | Description |
 | --- | --- | --- | --- |
-| `AccountingItems` | bool | optional | Whether the response should contain [Accounting item](#accounting-item)s. |
+| `OrderItems` | bool | optional | Whether the response should contain [Order item](#order-item)s. |
```

Comment:
Would it make sense to specify what is the default value?

Reply:
No, we don't want to commit to a default, rather to remove it.

Reply:
The Extent object itself is required, these will be interpreted as false if not provided, so technically optional. But get the reason to mark them as required anyway.